guides/source/configuring.md

```diff
@@ -997,7 +997,7 @@ You can find more detailed configuration options in the
 
 `config.active_storage` provides the following configuration options:
 
-* `config.active_storage.variant_processor` accepts a symbol `:mini_magick` or `:vips`, specifying whether variant transformations and blob analysis will be performed with MiniMagick or ruby-vips. The default is `:mini_magick`.
+* `config.active_storage.variant_processor` accepts a symbol `:vips` or `:mini_magick`, specifying whether variant transformations and blob analysis will be performed with MiniMagick or ruby-vips. The default is `:vips`.
```

Comment:
```suggestion
* `config.active_storage.variant_processor` accepts a symbol `:vips` or `:mini_magick`, specifying whether variant transformations and blob analysis will be performed with MiniMagick or ruby-vips. The default is `:mini_magick`.
```

This should match what's here: https://github.com/FestaLab/rails/blob/vips-as-default/activestorage/lib/active_storage.rb#L47

The rest is correct, ie setting it through `load_defaults` and `new_framework_deafults`.